linera-core/src/client/mod.rs

```diff
@@ -157,6 +157,9 @@ where
     message_policy: MessagePolicy,
     /// Whether to block on cross-chain message delivery.
     cross_chain_message_delivery: CrossChainMessageDelivery,
+    /// An additional delay, after reaching a quorum, to wait for additional validator signatures,
+    /// as a fraction of time taken to reach quorum. If `None`, uses the default value.
+    grace_period: Option<f64>,
```

Comment:
Note that we could remove the `Option` entirely by injecting the default value during CLI parsing (with or without moving the constant).

Reply:
That would work for the CLI.  Clients created in `linera-service` would also need to inject the default.
 
I've added a commit that shows what that looks like. This `linera-service` concern also shows why it may make sense to keep `DEFAULT_GRACE_PERIOD` in the core crate.